keyboards/varanidae/readme.md

```diff
@@ -0,0 +1,24 @@
+# Varanidae
+
+![Varanidae](https://i.imgur.com/0wWfmE1h.png)
+
+Varanidae the left-side pad 110 keys keyboard for creators powered by Elite-C.
+
+* Keyboard Maintainer: [Silc Renew](https://github.com/TokageItLab)
+* Hardware Supported: Varanidae
+* Hardware Availability: [TokageItLab / Varanidae](https://github.com/TokageItLab/Varanidae)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make varanidae:default
+    make varanidae:via
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
```

Comment:
You should write something more precise here — your (0,0) key is certainly not Esc (it appears to be F16 in the default keymap, and the top left location is occupied by an encoder).

Alternatively, you can redefine the bootmagic key to use another matrix location (including the key which you intend to be Esc).

Reply:
Replaced "Esc" with "F16 as default".

Reply:
Finally, the 0,0 element of the matrix became an encoder push switch so I changed the description to that.